thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -47,6 +47,8 @@ def __init__(self):
 
         self.output_to_zero()
 
+        rospy.on_shutdown(self.shutdown)
```

Comment:
Good idea. We can always change it to a separate hook later, but for now `self.output_to_zero()` is enough.